x-pack/plugins/encrypted_saved_objects/server/lib/encrypted_saved_objects_service.ts

```diff
@@ -0,0 +1,218 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+// @ts-ignore
+import nodeCrypto from '@elastic/node-crypto';
```

Comment:
__note:__ maybe it'd make sense to move `node-crypto` to Kibana's packages and typescriptify it...... It relies on kibana eslint config and babel preset already anyway.

Reply:
> Do you know if it's theoretically possible to publish a kbn package as a node module?

It's definitely possible since we do this for [@elastic/datemath](https://github.com/elastic/kibana/pull/26559) already so that [@elastic/eui can consume it](https://github.com/elastic/eui/blob/master/package.json#L80).

Even though it's not recommended I'd vote for moving these packages to Kibana and publish them to NPM to have such important packages integrated into Kibana's test/lint/build/audit/greenkeeper/docs/you-name-it processes.

Hey @elastic/kibana-platform, how do you feel about moving [@elastic/request-crypto](https://github.com/elastic/request-crypto) and [@elastic/node-crypto](https://github.com/elastic/node-crypto) to `kibana/packages` **and** publishing them to NPM from there (so that it can be consumed by Telemetry service and potentially other consumers within Elastic)?